README.md

```diff
@@ -29,7 +29,7 @@
   </p>
 </p>
 
-[![Home](docs/home.png)](http://weather_report.surge.sh/)
+[![Home](public/weather/home.png)](http://weather-report-israellaguan.vercel.app/)
```

Comment:
Consider moving to a `/docs` file instead, public should be to serve in production, but these images are just for the readme